docs/blog/welcome-new-oras-owners.md

```diff
@@ -0,0 +1,21 @@
+# ORAS Welcomes New Owners, Maintainers, and Thanks Emeritus Owners
+
+_ORAS Community_
+
+## Welcomes new owners and maintainers
+
+With a supermajority vote from the existing ORAS Organization Owners, we are excited to announce that [Terry Howe](https://github.com/TerryHowe) and [Andrew Block](https://github.com/sabre1041) have been accepted as new ORAS Organization owners. In addition, [Billy Zha](https://github.com/qweeah) and [Feynman Zhou](https://github.com/feynmanzhou) been accepted as ORAS subproject maintainers.
+
+Terry has been actively contributing to ORAS since Dec 2022 and has been nominated to a sub-project maintainer on Feb 21. He raised 19 PRs to multiple ORAS repositories includes ORAS CLI, ORAS-go, and ORAS-www in the last quarter. He also reviewed Pull Request actively and left 126 comments in the same period. 
+
+Andrew has been involved in the ORAS community since its inception as a Helm maintainer. He saw how the capabilities produced by ORAS could be used outside of the Helm ecosystem, such as in [Emporous](https://emporous.io) (formerly called Universal Object Reference UOR) as a way to manage content within OCI registries. He also contributed to several features and bug fixes for ORAS-go.
+
+Billy has contributed 103 PRs to the main branch and reviewed most the PRs of the ORAS CLI repository in the last 9 months. He also presented several demos and proposals in the ORAS community meeting. 
+
+Feynman has been actively contributing to the ORAS-www and ORAS CLI repositories in the past 10 months. He raised 11 Pull Requests and helped review several PRs in multiple repositories. He is also one of the meeting chair who organizes the ORAS community meeting in the last 10 months.
+
+They have demonstrated their commitment and passion for the project. We are grateful for their contributions and leadership, and we look forward to working with them as owners of ORAS.
+
+## Thanks Emeritus Owners
+
+We also want to thank [Avi Deitcher](https://github.com/deitch) and [Josh Dolitsky](https://github.com/jdolitsky) who have stepped down from their roles as owners of ORAS. Avi was one of the founding members of ORAS who helped shape its vision and direction. Josh was also one of the founding members of ORAS who instrumental in developing many features of ORAS such as artifact reference types, push/pull options, etc. They have both moved to emeritus status but will continue to support ORAS as advisors. We appreciate their dedication and service to the project, and we wish them all the best for their future endeavors.
```

Comment:
It would be good to 
```suggestion
We also want to thank [Avi Deitcher](https://github.com/deitch) and [Josh Dolitsky](https://github.com/jdolitsky) who have stepped down from their roles as owners of ORAS. Avi was one of the founding members of ORAS who helped shape its vision and direction. Josh was also one of the founding members of ORAS who instrumental in developing many features of ORAS, such as artifact reference types, push/pull options, etc. They have both moved to emeritus status but will continue to support ORAS as advisors. We appreciate their dedication and service to the project, and we wish them all the best for their future endeavors.
```